config/crd/bases/operator.gatekeeper.sh_gatekeepers.yaml

```diff
@@ -684,6 +684,11 @@ spec:
                     container image
                   type: string
               type: object
+            mutatingWebhook:
```

Comment:
How about calling this just "MutationEnabled" or something along that line?
Asking this because it does more than just deploying the webhook